pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -483,7 +483,7 @@ func TestDesiredRouterDeploymentSpecTemplate(t *testing.T) {
 			if volume.Secret.SecretName != secretName {
 				t.Errorf("router Deployment expected volume %s to have secret %s, got %s", volume.Name, secretName, volume.Secret.SecretName)
 			}
-		} else if volume.Name != "service-ca-bundle" {
+		} else if volume.Name != controller.IngressCAConfigMapName().Name {
```

Comment:
Any reason for the changes in this file?  For test code, I think it is appropriate to use explicit string literals.

Reply:
The volume name is changed to `ingress-ca-bundle` by this implementation and while updating the same thought to use the definition and avoid hard coding. Please let me know, if it needs to be reverted.